dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/AsyncMethodWrapper.java

```diff
@@ -166,105 +161,99 @@ public void run()
         // F743761.CodRv - Void methods do not throw application exceptions.
         s.ivIgnoreApplicationExceptions = isVoidReturnType;
 
-        try
-        {
-            try
-            {
-                Object theBean = container.EjbPreInvoke(this, ivMethodId, s, ivArgs);
+        AbstractEJBRuntime rt = (AbstractEJBRuntime) super.container.getEJBRuntime();
+
+        // If the server is stopping, don't invoke the async method
+        if (!rt.isStopping()) {
```

Comment:
A Tr.debug indicating asynch method is being skipped because server is stopping would be nice.

Reply:
Thanks.  Added a debug statement in the new commit.